tool_path_planner/src/surface_walk_raster_generator.cpp

```diff
@@ -1198,94 +1198,66 @@ vtkSmartPointer<vtkPolyData> SurfaceWalkRasterGenerator::createStartCurve()
   vtkSmartPointer<vtkCellArray> cell_ids = mesh_data_->GetPolys();
   cell_ids->InitTraversal();
 
-  double avg_center[3] = { 0, 0, 0 };
-  double avg_norm[3] = { 0, 0, 0 };
+  Eigen::Vector3d avg_center(Eigen::Vector3d::Zero());
+  Eigen::Vector3d avg_norm(Eigen::Vector3d::Zero());
+
   double avg_area = 0;
   long num_cells = cell_ids->GetNumberOfCells();
```

Comment:
Recommend using `{}` initialization see [here](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es23-prefer-the--initializer-syntax).

``` c++
Eigen::Vector3d avg_center { Eigen::Vector3d::Zero() };
Eigen::Vector3d avg_norm { Eigen::Vector3d::Zero() };

double avg_area {0};
long num_cells {cell_ids->GetNumberOfCells()};
```